.github/workflows/test.yml

```diff
@@ -25,32 +27,17 @@ jobs:
         DATABASE_PORT: 3306
         SECRET_KEY: ${{ secrets.SECRET_KEY }}
         DJANGO_SETTINGS_MODULE: answerking.settings.development
-    
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}   # Generate a token on Sonarcloud.io, add it to the secrets of this repo with the name SONAR_TOKEN (Settings > Secrets > Actions > add new repository secret)
     steps:
-      - uses : actions/checkout@v3
+      - uses : actions/checkout@v2
```

Comment:
why are we using checkout@v2 over v3?

Reply:
could we change it back to v3 now and check if it works. will just have been v2 due to when the example was written i suspect

Reply:
the v3 was merged in develop and the comments from the bot are not working, I want to give it a try with the v2